radicle-cli/src/commands/init.rs

```diff
@@ -35,6 +35,7 @@ Options
     --setup-signing      Setup the radicle key as a signing key for this repository
     --announce           Announce the new project to the network
     --no-confirm         Don't ask for confirmation during setup
+    --verbose, -v        Verbose mode
     --help               Print help
```

Comment:
For consistency, the shortened version should precede the longer one.  Or the other commands should be updated.

From `rad review`:
```
Options
    -r, --revision <number>   Revision number to review, defaults to the latest
        --[no-]sync           Sync review to seed (default: sync)
    -m, --message [<string>]  Provide a comment with the review (default: prompt)
        --no-message          Don't provide a comment with the review
        --help                Print help
```